CHANGES.md

```diff
@@ -138,6 +139,34 @@ OpenSSL 3.1
 
    *Tomáš Mráz*
 
+ * Various fixes and extensions to certificate handling and the `x509` etc. apps
+
+   *David von Oheimb*
+
+ * Various fixes and extensions to the CMP+CRMF implementation and the `cmp` app
+
+   *David von Oheimb*
+
+ * Fixes and extensions to the HTTP client and to the HTTP server in `apps/`
+
+   *David von Oheimb*
+
+ * Extended the CMS API for handling `CMS_SignedData` and `CMS_EnvelopedData`
+
+   *David von Oheimb*
+
+ * Moved coding style <https://www.openssl.org/policies/codingstyle.html>
+   to <https://www.openssl.org/policies/technical/coding-style.html>,
+   cleaned up the text, made some clarifications, and added various details.
+
+   *David von Oheimb et al.*
```

Comment:
This IMO does not belong here, it is not a code change in the openssl main repository.

Reply:
> This IMO does not belong here, it is not a code change in the openssl main repository.

I see. Where else to mention such (policy etc.) changes?

Reply:
I see no need to explicitly mention it anywhere. The policy changes are announced via the github PR merges in the technical policy repo and by the commit e-mails on the technical-policy repo.

Maybe the commit notification e-mails for these should go to openssl-project ML.

Reply:
I agree with @t8m, this part can just be omitted.